### ISSUE TITLE
Composition / folding function

### ISSUE THREAD

**Issue by jongold** [jongold/further#12]
So I was doing a lot of `s1.concat(s2).concat(s3).concat(s4).concat(s5)` vibes when I remembered that fold exists and that's literally one of the chillest things about monoids.

```js
import S from 'sanctuary';
import Style from '@jongold/further';

// fold :: Array Style -> Style
const fold = S.reduce(S.concat, Style.empty());

fold([s1, s2, s3])
```

presumably this should live in `further` _somewhere_. we just don't know _where_.

afaik Style isn't `foldable` itself, rather an array of Style is foldable.

does this make sense?
```js
.fold :: Array (Style a) -> Style a
Style.fold([s1,s2,s3])
```

should it be called `compose`? something else? I don't know 🙃

**Comment by jbaxleyiii** [comment]
`.fold` could be confusing becuase of https://github.com/fluture-js/Fluture#fold
`.compose` sounds good to me 🥇 because it is just compose(Style(fn), Style(fn)) just in array form?
`.from` as in from a collection of `.of`?

![lets do this](https://media.giphy.com/media/l4YoxVONrqqfS/giphy.gif)

**Comment by jbaxleyiii** [comment]
Implemented with #13
